tests/aem_hw.pm

```diff
@@ -393,33 +393,33 @@ sub run_cmd {
 
 sub run_scp_to_sut {
     my @cmd = ('sshpass', '-puserpass', 'scp', '-q', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null');
-    push @cmd, @_;
-    push @cmd, "root\@@{[ get_var('QUBES_OS_HOST_IP') ]}:";
+    push @cmd, @_[0];
+    push @cmd, "root\@@{[ get_var('QUBES_OS_HOST_IP') ]}:" . @_[1];
     return run_cmd(@cmd);
 }
 
 sub setup_acm {
+    ### FIXME: handle optiplex
     my $url = 'https://cdrdv2.intel.com/v1/dl/getContent/630744';
-    my $zip_root = '630744_003'; ### FIXME: update to new version, handle optiplex
+    my $zip_root = '630744_004';
     my $zip_fname = "$zip_root.zip";
 
-    my $bin_fname;
     if (check_var('MACHINE', 'optiplex')) {
-        $bin_fname = 'SNB_IVB_SINIT_20190708_PW.bin';
-    } else {
-        die "Don't know which ACM to use for '@{[ get_var('MACHINE') ]}' machine!";
+        die "Intel no longer provides SNB_IVB_SINIT_20190708_PW.bin file :(";
     }
 
     if (run_cmd('wget', '-O', $zip_fname, $url) != 0) {
         die "Failed to download '$url'";
     }
 
-    if (run_scp_to_sut($zip_fname) != 0) {
+    assert_script_run("mkdir -p '/tmp/$zip_root'");
+    if (run_scp_to_sut($zip_fname, "/tmp/$zip_root") != 0) {
         die "Failed to send ACM to DUT";
     }
 
+    assert_script_run("cd '/tmp/$zip_root'");
     assert_script_run("unzip -o '$zip_fname'");
-    assert_script_run("cp --update '$zip_root/$bin_fname' /boot");
+    assert_script_run("cp --update *.bin /boot");
```

Comment:
Copying all ACMs results in tests failure because GRUB waits for user's input after printing more lines that fit on the screen and pager is enabled.  Can either copy only the file we need or run `set pager=0` as a workaround.

Reply:
`set pager=0` is not enough. Because `slaunch_module` for incompatible ACMs returns error, GRUB still asks for confirmation before booting. I thought we improved it already, maybe it lives in some forgotten branch. Anyway, I'll modify the tests to copy just one ACM as it was done previously, and we can work on GRUB error handling later.

Reply:
Changed in https://github.com/3mdeb/openqa-tests-qubesos/pull/9/commits/e9d8ab5d5566effe11ae384cfcd3e70ba751d3c9, running tests now

Reply:
`Press any key to  continue...` shouldn't be a problem unless the entry is run after editing when [GRUB waits indefinitely](https://github.com/TrenchBoot/grub/blob/7259d55ffcf124e32eafb61aa381f9856e98a708/grub-core/normal/menu_entry.c#L1458-L1459), otherwise the boot continues [after a 10s delay](https://github.com/TrenchBoot/grub/blob/7259d55ffcf124e32eafb61aa381f9856e98a708/grub-core/normal/menu.c#L62-L74).  But I do like not having the screen filled with irrelevant errors about lots of ACMs, the only drawback is that this doesn't test handling of multiple Slaunch modules.